app/assets/stylesheets/modules/_landing.scss

```diff
@@ -0,0 +1,41 @@
+.landing {
+  .banner {
```

Comment:
I tend to avoid nesting css classes and follow the module rules in [this book](https://smacss.com/book/type-module).

I usually do something like `.landing-banner`, `.landing-heading`, `.landing-blurb`.

Would love to hear your thoughts too :)